examples/next/pages/ui/components/storage/storage-manager/process-file-access-level/index.page.tsx

```diff
@@ -20,7 +23,12 @@ const processFile: StorageManagerProps['processFile'] = async ({ file }) => {
       const hashHex = hashArray
         .map((a) => a.toString(16).padStart(2, '0'))
         .join('');
-      return { file, key: `${hashHex}.${fileExtension}` };
+      return {
+        file: new File([blob], `${hashHex}.${fileExtension}`, {
+          type: 'text/plain',
+        }),
```

Comment:
Why do we create a mock file while the original file's hash gets returned along with?